openapi.yml

```diff
@@ -109,6 +111,38 @@ paths:
             application/json:
               schema:
                 $ref: "#/components/schemas/Organization"
+  /organizations/{ipaCode}/documents/{documentName}:
+    get:
+      description: Returns the requested document
+      operationId: getDocument
+      security:
+        - bearerAuth: []
+        - tokenQueryAuth: []
```

Comment:
a question: is there any authentication mechanism that uses a token in the query ? (or we just use bearer tokens)

Reply:
This project uses the Bearer authentication, in all the authenticated calls we expect a token inside the `Authorization` header. Considering that this is the endpoint to download a PDF document, I thought it could be convenient to allow the authentication using a query parameter, so that the endpoint can be called by clicking an `<a>` tag embedded in the user page. This authentication mechanism using a query parameter is handled by the [passport-http-bearer](https://github.com/jaredhanson/passport-http-bearer#making-authenticated-requests) library used in this project.

Reply:
thank you for the insight ! there are some security concerns about tokens inside query params (see https://stackoverflow.com/a/643480/21424) so I'd avoid them if avoidable and/or the data is sensitive.